draft-dejong-remotestorage-06.txt

```diff
@@ -297,7 +299,7 @@ Internet-Draft              remoteStorage                  November 2015
        * silent deletion of the parent folder if it is left empty by
          this, and so on for further ancestor folders,
        * the version of its parent folder being updated, as well as that
-         of further ancestor folders.
+         of further ancestor folders if they are not deleted.
```

Comment:
That's *another* way to think about it, but I don't think it's possible to rephrase the entire RFC around the concept of folder listings being prefix-searches in a kv-store. Given that, we have to decide whether folders always exist, or whether they're automatically created and deleted. We can't claim both, though the effects of that phrasing error probably would be very modest.